src/View/FontsService.py

```diff
@@ -15,7 +15,7 @@ def __init__(self):
 		if(current_os == 'Windows'):
 			path_to_fonts += '\\src\\res\\fonts\\'
```

Comment:
Is this a missed reference still pointing to src?

Reply:
Good catch. I tried to a variety of combinations of paths to find uses of the paths. Will do another sweep through.